core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-list/dot-categories-list.component.scss

```diff
@@ -0,0 +1,23 @@
+@use "variables" as *;
+
+:host {
+    ::ng-deep {
+        .action-header__global-search {
+            flex-wrap: wrap;
+        }
+        .dot-category-link-button,
+        .dot-category-link-button:hover,
+        .dot-category-link-button:enabled,
+        .dot-category-link-button:enabled:hover {
+            color: var(--color-sec);
+        }
+        .container-listing__header-options,
+        .dot-category-line-break {
+            width: 100%;
+        }
```

Comment:
Are you sure we need this nested in the `::ng-deep`?

Reply:
updated

Reply:
updated